benefits/eligibility/templates/eligibility/includes/modal--contactless.html

```diff
@@ -3,7 +3,7 @@
 {% load static %}
 
 {% block modal-content %}
-    <h2 id="contactless-modal">{% translate "eligibility.pages.start.modal.title" %}</h2>
+    <h2 id="contactless-modal" class="me-4 me-md-0">{% translate "eligibility.pages.start.modal.title" %}</h2>
```

Comment:
What are the `me-4` and `me-md-0` classes here? I couldn't find that prefix `me` in the [Bootstrap docs](https://getbootstrap.com/docs/4.1/utilities/spacing/).

Reply:
So the margin/padding class naming logic is like this for Bootstrap 5.0 and beyond:

- `m` or `p`: margin or padding
- `t`: top
- `b`: bottom
- `y`: top and bottom
- `x`: left and right

and here's the new, Left to Right/Right to Left language-compatible friendly version of:
- `s`: short for START. for our languages (English and Spanish), that means `left`
- `e`: short for END. for our languages (English and Spanish), that means `right`

Documented here: https://getbootstrap.com/docs/5.3/utilities/spacing/#notation

Reply:
Actually this is the real link for our version: https://getbootstrap.com/docs/5.1/utilities/spacing/

You were referencing 4.1, but this app uses 5.1

Reply:
TL;DR - `e` means `end`. As in the end of the line for LTR text, so it's `right`.

Reply:
Doh, my bad on the version mismatch. Thanks!